src/App.js

```diff
@@ -1,10 +1,12 @@
 import React from "react";
 import "./styles.css";
+import PokeCard from "./components/PokeCard";
 
 export default function App() {
   return (
     <div className="App">
       <h1>Welcome to your second mini challenge!</h1>
+      <PokeCard />
```

Comment:
better if you load mock data out of the component and then pass those values as component properties